Assets/XML/Text/Buildings.xml

```diff
@@ -720,7 +720,11 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_BABYLONIAN_ZIGGURAT_STRATEGY</Tag>
-		<English>The [COLOR_BUILDING_TEXT]Ziggurat[COLOR_REVERT], the Babylonian replacement for the [COLOR_BUILDING_TEXT]Pagan Temple[COLOR_REVERT], requires the [COLOR_TECH_TEXT]Polytheism[COLOR_REVERT] technology, doesn't require the [COLOR_TECH_TEXT]Mysticism[COLOR_REVERT] technology, provides extra culture and gives a small science and culture boost.</English>
+		<English>The [COLOR_BUILDING_TEXT]Ziggurat[COLOR_REVERT] replaces the [COLOR_BUILDING_TEXT]Pagan Temple[COLOR_REVERT] for civilizations that follow [COLOR_HIGHLIGHT_TEXT]Anunnaki[COLOR_REVERT] and originate in Mesopotamia.</English>
```

Comment:
I think I would prefer a formulation along the lines of "The Ziggurat is the Pagan Temple of civilizations that follow Anunnaki and ..."